### PR TITLE
Features/create footer

### PR DESCRIPTION
**Pull request by marychris**
Add footer markup.
Theme footer.
Add new image camp-logo-footer.png.
Add fontawesome library.

Note: The href value for  'terms of use', 'code of conduct', and 'access the source the code' is just #, we probably need a value.

### CODE REVIEW

**Comment by Luukyb** [comment]
could we use the svg file we already have instead of the static/images/camp-logo-footer.png ?

File: static/index.html [hunk 1]
```diff
@@ -61,6 +62,15 @@
       </div>
     </div>
   </div>
+  <footer>
+    <div class="container">
+      <div class="col-md-12">
+        <div class="camp-logo-footer"><img src="images/camp-logo-footer.png" /></div>
```
Comment (Luukyb) [inline static/index.html]: please comment.

File: static/index.html [hunk 2]
```diff
@@ -61,6 +62,15 @@
...
+        <p>Since 2014, Drupalcamp Cebu is brought to you by the <a href="https://www.facebook.com/groups/drupalcebu/">Drupal Cebu User Group.</a> <a href="#">Terms of use.</a> <a href="#">Code of conduct.</a></p>
```
Comment (Luukyb) [inline static/index.html]: facebook link in a new target please :)

File: static/index.html [hunk 3]
```diff
@@ -4,6 +4,7 @@
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
   <link rel="stylesheet" href="stylesheets/style.css">
   <link href="https://fonts.googleapis.com/css?family=Muli:200,400|Saira:400,700" rel="stylesheet">
+  <link rel="stylesheet" href="lib/font-awesome/css/font-awesome.min.css">
```
Comment (Luukyb) [inline static/index.html]: Let's use the cdn version of font awesome. so we don't need to store it, and users have a better performance.

Comment (marychris) [reply static/index.html]: Done

**Comment by Luukyb** [comment]
thanks!

**Pull request merged**